### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ongmin**
:sunflower:

### CODE REVIEW

**Comment by ongmin** [comment]
Maybe we should update into Master?

**Comment by caalberts** [comment]
👆🏼👈🏼👇🏼👉🏼 👍🏼

**Comment by ongmin** [comment]
Urm.. Disco fever?

File: app/components/channel/playlist.js [hunk 1]
```diff
@@ -61,12 +31,19 @@ var VideoItem = React.createClass({
...
 var VideoTable = React.createClass({
   propTypes: {
-    videos: React.PropTypes.array.isRequired
+    videos: React.PropTypes.array.isRequired,
+    playVideo: React.PropTypes.func
+  },
+
+  componentDidMount: function () {
+    var container = React.findDOMNode(this)
+    dragula([container])
```
Comment (caalberts) [inline app/components/channel/playlist.js]: This turns on drag and drop. Now that the playlist plays continuously, the drop action needs to update the playlist array in store to keep the store playlist consistent with what is rendered

**Comment by caalberts** [comment]
refer to #33

**Pull request closed without merging**